src/_js/main.mjs

```diff
@@ -42,6 +59,14 @@ if (location.search.includes('utm_source')) {
   history.replaceState({}, '', location.pathname);
 }
 
+// Helper function to dynamically insert scripts.
+const firstScript = document.scripts[0];
+const insertScript = (src) => {
+  const scriptElement = document.createElement('script');
+  scriptElement.src = src;
+  firstScript.parentNode.insertBefore(scriptElement, firstScript);
```

Comment:
Why do we always want to insert it as the first script instead of `document.head.appendChild(...)`?

Reply:
The difference doesn’t matter in practice anymore these days. I’d be fine with changing it!